specification/storage/data-plane/Microsoft.StorageDataLake/stable/2020-06-12/DataLakeStorage.json

```diff
@@ -1532,6 +1639,11 @@
                 "x-ms-client-name": "Continuation",
                 "description": "When deleting a directory, the number of paths that are deleted with each invocation is limited.  If the number of paths to be deleted exceeds this limit, a continuation token is returned in this response header.  When a continuation token is returned in the response, it must be specified in a subsequent invocation of the delete operation to continue deleting the directory.",
```

Comment:
but out of curiosity. Do we follow continuation when deleting directory in the SDK? this kind of looks similar to recursive acl.

I wonder how deletion-id behaves if there's continuation. (if this is even a case)

Reply:
We do not, the continuation only applies to non-HNS enabled accounts.  We only support HNS account with the Data Lake SDK.